### ISSUE TITLE
Recent Binaryen crashes when memory is importing ("--importMemory")

### ISSUE THREAD

**Issue by MaxGraey** [AssemblyScript/assemblyscript#2482]
Depeds on https://github.com/WebAssembly/binaryen/issues/4988

**Comment by RefinedSoftwareLLC** [comment]
```
{
  "imports": {
    "binaryen": "https://cdn.jsdelivr.net/npm/binaryen@109.0.0-nightly.20220818/index.js",
    "long": "https://cdn.jsdelivr.net/npm/long@5.2.0/index.js",
    "assemblyscript": "https://cdn.jsdelivr.net/npm/assemblyscript@0.21.1/dist/assemblyscript.js",
    "assemblyscript/asc": "https://cdn.jsdelivr.net/npm/assemblyscript@0.21.1/dist/asc.js"
  }
}
```
These are the highest versions I could get to work together.

**Comment by MaxGraey** [comment]
Fix will available with upcoming v0.21.3
